YellowNotes.Api/Startup.cs

```diff
@@ -22,6 +22,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             services.AddDbContextPool<DatabaseContext>(options => 
                 options.UseMySql(Configuration.GetConnectionString("Development")));
+            services.AddTransient<YellowNotes.Core.Services.IEmailService, YellowNotes.Core.Services.EmailService>();
```

Comment:
Tutaj daj `using YellowNotes.Core.Services;`, to nie trzeba będzie pisać tych długich namespace'ów.